main/res/values/strings.xml

```diff
@@ -1507,7 +1507,7 @@
     <string name="waypoint_cache_coordinates">Cache coordinates</string>
     <string name="waypoint_calculate_coordinates">Calculate coordinates</string> <!-- Legacy: button text in coordinate input -->
     <string name="waypoint_calculated_coordinates">Calculated coordinates</string> <!-- Legacy: line in waypoint description -->
```

Comment:
Is this one and the one above still needed?

Reply:
@Lineflyer there is a whole lot of code in c:geo which will no longer be needed after the switch. However, I want to keep it for now and see if things stay stable for at least one release. Then I will remove all that code. 

I did the same thing with filtering, and I find this to be a usefuil pattern. To not forget about the removal I created issue #12380 a while ago. I also collect there the things which need removal.

Reply:
That said, the two lines mentioned by you may create an "unused resource" warning. Have to check that.

Reply:
Removed one, kept the other (which is still used)